corehq/apps/fixtures/fixturegenerators.py

```diff
@@ -49,6 +49,32 @@ def item_lists_by_domain(domain):
     return ret
 
 
+def item_lists_by_app(app):
+    LOOKUP_TABLE_FIXTURE = 'lookup_table_fixture'
+    REPORT_FIXTURE = 'report_fixture'
+    lookup_lists = item_lists_by_domain(app.domain).copy()
+    for item in lookup_lists:
+        item['fixture_type'] = LOOKUP_TABLE_FIXTURE
+
+    report_configs = [
+        report_config
+        for module in app.get_report_modules()
+        for report_config in module.report_configs
+    ]
+    ret = list()
+    for config in report_configs:
+        uri = 'jr://fixture/commcare-reports:%s' % (config.uuid)
+        ret.append({
+            'id': config.uuid,
+            'uri': uri,
+            'path': "/rows/row",
+            'name': config.header.get('en'),
```

Comment:
Since you have the app, can you use the default language here instead of English?

Reply:
Sure, added.